manifest/net.kazu0617/SessionTweaks/info.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "SessionTweaks",
+	"id": "net.kazu0617.sessiontweaks",
+	"description": "Add these 3 buttons, 'Spawn' as spawn session orb, 'Open' as join session without focusing' and 'Copy' as copies session uri in Contacts tab.",
+	"category": "Misc",
```

Comment:
Seems to fall under the "Dashboard" category pretty well.

Reply:
Thanks for the suggestion. updated it.